CONTRIBUTING.md

```diff
@@ -1,5 +1,29 @@
-
-# Developer's Certificate of Origin 1.1
+# Contributing guide
+
+## Building the guide locally
+
+```bash
+# Clone this repository first, and then switch to the project directory
+cd package-examples
+cd guide
+npm ci
+npm run build  # Build the guide book in _book
+npm start      # Run a development server that host the built guide
+```
```

Comment:
Just running `npm start` was enough for me.

Reply:
The build command is not necessary for starting the dev server, but is useful on its own if a dev server is not necessary.